lib/notifier.js

```diff
@@ -0,0 +1,54 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+/**
+ * This notifier is called by the logger via `notifyAttachedServices`
+ * to send notifications to Amazon SNS/SQS.
+ */
+const AWS = require('aws-sdk')
+const config = require('../config')
+
+const snsTopicArn = config.get('snsTopicArn')
```

Comment:
The config value name `snsTopicArn` doesn't mention what the topic is used for. Maybe `notifierSnsTopicArn`?